portfolio/src/main/java/com/google/sps/servlets/CommentHandlerServlet.java

```diff
@@ -5,24 +5,22 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.sps.apihelper.DatastoreHelper;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Safelist;
 
 @WebServlet("/suggestions")
 public class CommentHandlerServlet extends HttpServlet {
 
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         
-        //gets comment text
-        String comment = request.getParameter("text-comment");
+        //gets information
+        String comment = Jsoup.clean(request.getParameter("text-comment"), Safelist.none());
         String name = request.getParameter("commenter-name");
```

Comment:
Good idea. I will say that as of now the name isn't displayed intentionally. Is it still recommended to clean it incase I decide to use it later. Especially since old data would still be stored in an unclean state even after I decide to clean it and use it.